src/lib/logger/vercel/vercelLogger.test.ts

```
@@ -344,60 +350,56 @@ describe("Comprehensive MaximVercelTracer Tests", () => {
 							traceId: traceId,
 							spanId: spanId,
-						}
-					}
+						},
+					},
 				});
 
 				const { object } = await generateObject({
 					model: model,
-					prompt: 'Extract the desired information from this text: \n' + rawOutput,
+					prompt: "Extract the desired information from this text: \n" + rawOutput,
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)